source/api_c/include/c_api.h

```diff
@@ -25,10 +25,42 @@ extern DP_Nlist* DP_NewNlist(int inum_,
                              int* numneigh_,
                              int** firstneigh_);
 
-/**
- * @brief Delete a neighbor list.
- *
- * @param nl Neighbor list to delete.
+extern DP_Nlist* DP_NewNlist_comm(int inum_,
+                                  int* ilist_,
+                                  int* numneigh_,
+                                  int** firstneigh_,
+                                  int nswap,
+                                  int* sendnum,
+                                  int* recvnum,
+                                  int* firstrecv,
+                                  int** sendlist,
+                                  int* sendproc,
+                                  int* recvproc,
+                                  void* world);
+
+/*
```

Comment:
comment should be written above the function. Here is for the function below

https://www.doxygen.nl/manual/docblocks.html